en/tutorials-and-examples/blog/part-three.rst

```diff
@@ -176,8 +176,12 @@ read if you want re-familiarize yourself with how CakePHP works.
 .. note::
     If you are on Windows remember to use \ instead of /.
 
+<<<<<<< HEAD
+You'll need to edit the following in **src/Template/Categories/add.ctp**::
```

Comment:
@mtkocak You can just keep

    You'll need to edit the following in **src/Template/Categories/add.ctp**
    and **src/Template/Categories/edit.ctp**::